sites/public/src/pages/404.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useContext } from "react"
```

Comment:
What prompted this addition?

Reply:
A warning popped up when building that no 404 page exists and they recommend creating one.. I think we can re-look at how we are handling errors in general.